girder/models/upload.py

```diff
@@ -243,6 +243,9 @@ def finalizeUpload(self, upload, assetstore=None):
         if '_id' in upload:
             self.remove(upload)
 
+        logger.info('Upload complete. Upload=%s File=%s User=%s' % (
+            upload['_id'], file['_id'], upload['userId']))
```

Comment:
Should we also add the size and/or the name of the file?  When trying to determine files of interest, these could help.

Reply:
> I didn't include any actual metadata about the file itself in the log, it seems more useful to just retain its ID and we could lookup the full metadata from the database via a script if needed.

Since much of the metadata is mutable, I figured we would be better off just using this as a way to do lookups into the database. Probably any analysis we'll want to perform will involve data munging from disparate sources.